### PR TITLE
Remove all items

### PR DESCRIPTION
**Pull request by ovekyc**
remove all items

There is an issue in test code.
The code go well in case of success test
However, when I forced code to flow error(for test) like change 128 line of src/test/node/items/controllers.js  
to  
if(!err2 && ...

It show error 

not ok 6 item2 is not removed

---

```
operator: fail
at: dispatchError (/Users/chan/WebstormProjects/goober/node_modules/levelup/lib/util.js:64:35)
```

  ...
and it occur change tape function like fail, equal or somthing

But i dont know what it is. I think it is kind of internal error of tape or levelup
help me please

### CODE REVIEW

**Comment by fguy** [comment]
👍 LGTM

File: src/test/node/items/controllers.js [hunk 1]
```diff
@@ -95,3 +95,52 @@ test('delete an item from database', t => {
...
+      url: '/items'
+    });
+
+    const res = httpMocks.createResponse();
+
+    ItemController.removeAll(req, res, () => {
+      const status = res.statusCode;
+      const message = res._getData().message;
+      testDB.get('item1', (err1) => {
+        if (err1 && err1.notFound) {
```
Comment (festinalente91) [inline src/test/node/items/controllers.js:126]: there could be a database internal error. 
[https://github.com/Level/levelup#dbgetkey-options-callback](https://github.com/Level/levelup#dbgetkey-options-callback)
Later we should talk about this issue.

**Comment by festinalente91** [comment]
LGTM

**Comment by fguy** [comment]
LGTM

File: src/server/items/controllers.js [hunk 2]
```diff
@@ -30,5 +30,32 @@ export default {
       }
       cb();
     });
+  },
+
+  removeAll: (req, res, cb) => {
+    const errorList = [];
+    db.createReadStream({
+      start: 'item1',
+      end: 'item' + '\xFF'
```
Comment (hewonjeong) [inline src/server/items/controllers.js:39]: We are using 'itemIncrement' to count number of items in DB.
It will remove itemIncrement variable too.

**Pull request merged**